activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,23 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.glob_to_exts(glob)
+        match = glob.match(/\{([\S]+)\}/)
+        match[1].split(",")
+      end
+
+      def self.compile_watched_dirs(paths)
```

Comment:
I have no idea what this means and why we can't use methods on `Rails::Paths::Path`. @kirs, can you elaborate? :smile: